src/containers/CreatePipelineRun/CreatePipelineRun.js

```diff
@@ -118,7 +126,7 @@ function CreatePipelineRun({ intl }) {
     setState
   ] = useState({
     ...initialState,
-    namespace: defaultNamespace !== ALL_NAMESPACES ? defaultNamespace : '',
+    namespace: getNamespace() || '',
```

Comment:
Is the OR still necessary here? Seems like its handled in the getNamespace function now

Reply:
good point, it would only be needed if defaultNamespace was falsy which can't happen. Testing the change now

Reply:
done 👍